setup.py

```diff
@@ -45,7 +38,7 @@ def get_version():
       license='Mozilla Public License 2.0 (MPL 2.0)',
       packages=['mozdownload'],
       zip_safe=False,
-      install_requires=deps,
+      install_requires=read('requirements.txt').splitlines(),
```

Comment:
Hurray that we already have the `read` method, but I think that we are doing bad in returning `None` in case of IOErrors. I think we should not catch this exception but let it bubble up. Otherwise the installation of mozdownload will be busted with a very strange failure.

Also please try to create a new mozdownload sdist package locally and check if the new file is part of it. Otherwise we would have to add it to a `MANIFEST.in` file.

Reply:
Thanks for the feedback. I will look into it a bit further. Also comparing it to FoxPuppet